projects/packages/wpcom-posts/composer.json

```
@@ -0,0 +1,54 @@
+{
+	"name": "automattic/jetpack-wpcom-posts",
+	"description": "Enhancements for the \"Posts\" screens on WordPress.com sites.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.1",
+		"automattic/wordbless": "dev-master"
+	},
+	"autoload": {
+		"files": [
+			"actions.php"
+		],
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"phpunit": [
+			"@composer update",
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"test-coverage": [
+			"@composer update",
+			"phpdbg -d memory_limit=2048M -d max_execution_time=900 -qrr ./vendor/bin/phpunit --coverage-clover \"$COVERAGE_DIR/clover.xml\""
+		],
+		"test-php": [
+			"@composer phpunit"
+		],
+		"post-update-cmd": "php -r \"copy('vendor/automattic/wordbless/src/dbless-wpdb.php', 'wordpress/wp-content/db.php');\""
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"mirror-repo": "Automattic/jetpack-wpcom-posts",
```

I think it would be okay to enable autotagger for this package.

```suggestion
		"autotagger": true,
		"mirror-repo": "Automattic/jetpack-wpcom-posts",
```

---

@sdixon194 Do you think we could add this to the `generate` command?

---

Sure thing, we can do that! Opened an issue to track it here: https://github.com/Automattic/jetpack/issues/19816